node/grpc/utils.go

```diff
@@ -59,12 +59,16 @@ func GetBlobMessages(in *pb.StoreChunksRequest, numWorkers int) ([]*core.BlobMes
 			for j, bundle := range blob.GetBundles() {
 				quorumID := blob.GetHeader().GetQuorumHeaders()[j].GetQuorumId()
 				if format == core.GnarkBundleEncodingFormat {
-					bundleMsg, err := new(core.Bundle).Deserialize(bundle.GetBundle())
-					if err != nil {
-						resultChan <- err
-						return
+					if len(bundle.GetBundle()) > 0 {
+						bundleMsg, err := new(core.Bundle).Deserialize(bundle.GetBundle())
+						if err != nil {
+							resultChan <- err
+							return
+						}
+						bundles[uint8(quorumID)] = bundleMsg
+					} else {
+						bundles[uint8(quorumID)] = make([]*encoding.Frame, 0)
```

Comment:
good catch. I think it is handling the case, when there is quorum which the operator does not belong to. But I thought you already catch it here, https://github.com/Layr-Labs/eigenda/blob/master/disperser/batcher/grpc/dispatcher.go#L376

Reply:
I guess the question is if the grpc would return a nil, or empty array.

Reply:
If the operator is not in this quorum, then the bundle is an empty []byte (not nil), so len(bundle.GetBundle()) is 0